src/statusBar/index.ts

```diff
@@ -101,16 +109,31 @@ export async function createCmdsStatusBarItems(workspaceFolder: Uri) {
     );
   }
 
-  if (projectConf) {
-    statusBarItems["projectConf"] = createStatusBarItem(
-      `$(${
-        commandDictionary[CommandKeys.SelectProjectConfiguration].iconId
-      }) ${projectConf}`,
-      commandDictionary[CommandKeys.SelectProjectConfiguration].tooltip,
-      CommandKeys.SelectProjectConfiguration,
-      99,
-      commandDictionary[CommandKeys.SelectProjectConfiguration].checkboxState
-    );
+  if (projectConfExists) {
+    if (!projectConf) {
```

Comment:
`projectConf` doesn't mean is an error. Means the user has not selected a profile at the moment. Even if `ESP.ProjectConfiguration.PROJECT_CONFIGURATION_FILENAME` exists.